### PR TITLE
Update n8n-nodes-base.actionnetwork.md

### PR DESCRIPTION
**Pull request by tiseysoft**
Updated Action network node page with the new intro. @StarfallProjects please review.

### CODE REVIEW

**Comment by StarfallProjects** [comment]
Hi @tiseysoft, thanks for this. Please can you sign the CLA?

File: docs/integrations/builtin/app-nodes/n8n-nodes-base.actionnetwork.md [hunk 1]
```diff
@@ -2,8 +2,13 @@
...
 [Action Network](https://actionnetwork.org/) is an open platform that empowers individuals and groups to organize for progressive causes.
 
+The Action Network node allows you to automate work in Action Network and integrate Action Network with other applications. n8n has built-in support for a wide range of Action Network features, including creating new events, adding new people, tags, and signatures. The Action Network node allows you to create, get, update, and delete events, petitions, attendance, person, and tags in your Action Network account. 
+
 !!! note "Credentials"
-    You can find authentication information for this node [here](/integrations/builtin/credentials/actionnetwork/).
+    Refer to the [Action Network credentials](/integrations/builtin/credentials/actionnetwork/) for guidance on setting up authentication. 
+
+!!! note "Examples and templates"
+    For example, usage and templates to help you get started, take a look at n8n's [Action Network integrations](https://docs.n8n.io/integrations/builtin/credentials/actionnetwork/) list.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/app-nodes/n8n-nodes-base.actionnetwork.md:11]: This should link to the integrations list, not the credentials page. Please clarify with Polina where to find these lists.

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
Looks good, just a couple of comments about links

**Comment by tiseysoft** [comment]
I have updated the Action Network link to open in new tab, and updated the integration links.

File: docs/integrations/builtin/app-nodes/n8n-nodes-base.actionnetwork.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+The [Action Network](https://actionnetwork.org/){:target="_blank" .external-link} node allows you to automate work in Action Network and integrate Action Network with other applications. n8n has built-in support for a wide range of Action Network features, which includes creating, updating, and deleting events, people, tags, and signatures. 
+
+On this page, you'll find a list of operations the Action Network node supports, and links to more resources.
 
 !!! note "Credentials"
-    You can find authentication information for this node [here](/integrations/builtin/credentials/actionnetwork/).
+    Refer to the [Action Network credentials](/integrations/builtin/credentials/actionnetwork/) for guidance on setting up authentication. 
+
+!!! note "Examples and templates"
+    For example, usage and templates to help you get started, take a look at n8n's [Action Network integrations](https://n8n.io/integrations/action-network/) list.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/app-nodes/n8n-nodes-base.actionnetwork.md:13]: This should use the external link format, as it goes out of the docs to the main website.

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
One more link change

**Comment by StarfallProjects** [comment]
Hi @tiseysoft 

This is amazingly fast progress! 🎉 

Can you go through and check/correct three things. These apply to multiple pages, so please check you have fixed all occurrences.

- The opening paragraph should start "The nodename node", not "Nodename node". Examples: [correct](https://deploy-preview-1081--n8n-docs.netlify.app/integrations/builtin/app-nodes/n8n-nodes-base.actionnetwork/) | [incorrect](https://deploy-preview-1081--n8n-docs.netlify.app/integrations/builtin/app-nodes/n8n-nodes-base.activecampaign/)
- The "Examples and templates" note should be "For example usage and templates", not "For example, usage and templates". I can see why you added the comma though. Perhaps "For usage examples and templates" would be even clearer? @ppolinaua what do you think?
- On some pages, the note isn't formatted correctly. For example: https://deploy-preview-1081--n8n-docs.netlify.app/integrations/builtin/app-nodes/n8n-nodes-base.affinity/

**Comment by StarfallProjects** [comment]
@tiseysoft in some places you're using & instead of 'and'. Please don't do this. Please can you replace all occurrences of '&' with 'and'.

**Comment by ppolinaua** [comment]
@tiseysoft Thanks a lot for handling our edits so patiently and making progress every day!

We've noticed you had introduced new phrasing that I don't think was in our template: instead of saying "n8n has built-in support for a wide range of <node-name> features, including <list of operations>", you're saying  "n8n has built-in support for a wide range of <node name> features, which includes **_basic operations_** like <list of operations>"

I'm guessing you got "basic operations" from the title in the doc. I know it seems minor, but we don't really want to use the phrase 'basic operations' and want to stick to the templated we agreed on. How about making edits and removing 'basic operations'? So the example would look like: 'n8n has built-in support for a wide range of Airtable features, including creating new events, and adding new people, tags, and signatures.' 

CC'ing @StarfallProjects

**Comment by tiseysoft** [comment]
> @tiseysoft And another slight detail about & sign: I opened around 30 PRs and spotted only one case like this: [f0befe3](https://github.com/n8n-io/n8n-docs/commit/f0befe34cd23666dd7c559b63702e2cbcf09b365) that says still ‘updating & deleting accounts’. Are you sure you removed & from all the nodes? Just double checking

Thank you both for these insightful feedbacks. I will edit accordingly right away!

**Comment by tiseysoft** [comment]
> > @tiseysoft And another slight detail about & sign: I opened around 30 PRs and spotted only one case like this: [f0befe3](https://github.com/n8n-io/n8n-docs/commit/f0befe34cd23666dd7c559b63702e2cbcf09b365) that says still ‘updating & deleting accounts’. Are you sure you removed & from all the nodes? Just double checking
> 
> Thank you both for these insightful feedbacks. I will edit accordingly right away!

Hello @StarfallProjects and @ppolinaua I have completed the edits that was asked of me. Please review. Thank you once again for always providing detailed and insightful feedbacks.

**Comment by ppolinaua** [comment]
Hey @tiseysoft Thanks for the updates!
Find below the list of typos I noticed - would be good to get them fixed:

General remarks:

- Refer to [Asana credentials](https://docs.n8n.io/integrations/builtin/credentials/asana/) for … (not ‘Refer to the [Asana credentials](https://docs.n8n.io/integrations/builtin/credentials/asana/) for …) - skip THE

- “The Affinity node allows you to automate work in Affinity, and integrate Affinity with other applications" (skip ‘the platform’ - not "The Affinity node allows you to automate work in the Affinity platform, and integrate Affinity with other applications”; add coma after ‘to automate work in Affinity,’)
For example here you did it correct:
https://github.com/n8n-io/n8n-docs/pull/1081/commits/f3a8ce461439566725ebcfd5d6874e81f55ec6cd
Just make sure all the intros are like this one

- Why no intros for these? [AMQP Sender](https://docs.n8n.io/integrations/builtin/app-nodes/n8n-nodes-base.amqp/), [APITemplate.io](http://apitemplate.io/), Asana?

Specific remarks: 

Agile CRM not Agility CRM
https://github.com/n8n-io/n8n-docs/pull/1081/commits/ea81be76221df5a52fb63df34edca76bfb51bb48

Airtable links should be here instead:
https://github.com/n8n-io/n8n-docs/pull/1081/commits/bf718c1e793486bffb9c41d247c77a0c727151c3 

Full stop (.) missing here after the 1st sentence:
https://github.com/n8n-io/n8n-docs/pull/1081/commits/deb44af2aaba3798f1e29530ba16bb812a2ad07f
https://github.com/n8n-io/n8n-docs/pull/1081/commits/044245233663d2c95b9273a6bb4cb5111470a200
https://github.com/n8n-io/n8n-docs/pull/1081/commits/8961ebbf8c762ceb3af9bc2d0620caf1358ab828 
https://github.com/n8n-io/n8n-docs/pull/1081/commits/1f05a332e4f25b44a5a03d11aa68a755db58dadb 

https://github.com/n8n-io/n8n-docs/pull/1081/commits/73132d77404933714756a7cb1b12887fe12f3cfb
take a look at n8n's [AWS Certificate Manager](https://docs.n8n.io/integrations/builtin/credentials/aws/){:target=_blank .external-link} integrations list.
> a wrong link here, should be this one instead:
https://n8n.io/integrations/aws-certificate-manager/

AWS SQS should be used instead of SNS
https://github.com/n8n-io/n8n-docs/pull/1081/commits/4a4eb70c0dfd2f66f7ac2f7405f4506de93b85ee

Beeminder not Beereminder 
https://github.com/n8n-io/n8n-docs/pull/1081/commits/6e2d33fcc89e8de0eaaed91050faca7c8a0c1a35

Bitwarden credetialS (S missing):
https://github.com/n8n-io/n8n-docs/pull/1081/commits/54c6fffe6efca371e57992eb4dd9d09b3380d329  

The Chargebee node allows… (The missing)
https://github.com/n8n-io/n8n-docs/pull/1081/commits/bf42d547fcce59d872214e36b31a14f1e091b716 
The missing also:
https://github.com/n8n-io/n8n-docs/pull/1081/commits/a44fb790bb183a3ec9711e81fdf5f2b6ad0bc7d9 

Click intro has Clearbit’s info:
https://github.com/n8n-io/n8n-docs/pull/1081/commits/60f588044abf53f8415e32348e69d2dac25bd54c 

Coda mentioned instead of CoinGecko
https://github.com/n8n-io/n8n-docs/pull/1081/commits/48defb752eacbe8ea265b95627de40cd6cdb2d6c

Convertkit has all info about Copper
https://github.com/n8n-io/n8n-docs/pull/1081/commits/d85b2ee0cd323ec26162f018c6d8d130c522da45 

Typo - should be Emelia instead of Drift
https://github.com/n8n-io/n8n-docs/pull/1081/commits/ad0051d13b977c2c72383489dfaf84d4a9fdfa2c
https://github.com/n8n-io/n8n-docs/pull/1081/commits/3968f8461fcb663056dbb47ee4de2dbe2e6093c9 
https://github.com/n8n-io/n8n-docs/pull/1081/commits/600660d39207708af0a273182654a8daab011c11 
https://github.com/n8n-io/n8n-docs/pull/1081/commits/8be2e0119aff1026cc6fff6c2296c05ae9444691 

Disqus instead of Hacker news - should not be so:
https://github.com/n8n-io/n8n-docs/pull/1081/commits/e646ffed7553cf24f928f3bfd413b58d2b1fd7a3

**Comment by ppolinaua** [comment]
Hey @tiseysoft A few more comments still, could you please address those?

I still don’t see Airtable links here:
https://github.com/n8n-io/n8n-docs/pull/1081/commits/8d068e054db86b239f33309d6c6bf7b3c8290131

Bitwarden credetialS (S missing): 
https://github.com/n8n-io/n8n-docs/pull/1081/commits/726ace749e636cdd143f5e78d298ccc71fc5d33c

The Chargebee node allows… (The missing)
[bf42d54](https://github.com/n8n-io/n8n-docs/commit/bf42d547fcce59d872214e36b31a14f1e091b716)
The missing also:
[a44fb79](https://github.com/n8n-io/n8n-docs/commit/a44fb790bb183a3ec9711e81fdf5f2b6ad0bc7d9)

Full stop (.) is missing after ‘goals’:
https://github.com/n8n-io/n8n-docs/pull/1081/commits/baee1ecdddcbd1a26d33faab0c749495b063fb9c

**Comment by ppolinaua** [comment]
@tiseysoft Hey! Also noticed:

- APITemplate.io instead of just APITemplate
docs/integrations/builtin/app-nodes/n8n-nodes-base.apitemplateio.md

- Rows should not be capitalized and full stop after it:
 docs/integrations/builtin/app-nodes/n8n-nodes-base.baserow.md

- Refer to [Beeminder credentials]( NOT to the)
docs/integrations/builtin/app-nodes/n8n-nodes-base.beeminder.md

-  'and integrate Drift' should be Gotify instead:
docs/integrations/builtin/app-nodes/n8n-nodes-base.gotify.md

**Pull request merged**